exasol_sagemaker_extension/autopilot_handler.py

```diff
@@ -44,4 +48,23 @@ def check_training_status(job_name: str):
         describe_response["AutoMLJobSecondaryStatus"]
 
 
+def deploy_endpoint(
+        job_name: str, instance_type: str, instance_count: int):
+
+    unique_endpoint_name = job_name  # endpoint_name = job_name
+    automl = AutoML.attach(auto_ml_job_name=job_name)
+    best_candidate = automl.describe_auto_ml_job(job_name)["BestCandidate"]
+    automl.deploy(
+        initial_instance_count=instance_count,
+        instance_type=instance_type,
+        candidate=best_candidate,
+        wait=True,
+        endpoint_name=unique_endpoint_name
+    )
+    return unique_endpoint_name
+
 
+def delete_endpoint(endpoint_name: str):
+    predictor = Predictor(endpoint_name)
+    predictor.delete_model()
```

Comment:
Does this delete the candidate or what does it delete?

Reply:
it deletes the deployed model located under "Models" section in Sagemaker web page. As far as I see, delete_endpoint() command also delete it, but to be sure I run delete_model() beforehand. (btw. It does not delete the trained job)